src/GenFacades.Core.Desktop/GenFacades.Core.Desktop.csproj

```diff
@@ -0,0 +1,40 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
```

Comment:
I wasn't implying we use the exe project. I'm OK having the wrapper exe separate from the dll I was more interested if we can eliminate the 2 builds of the dll.